lib/pacemaker/pcmk_verify.c

```diff
@@ -94,12 +93,15 @@ pcmk__verify(pcmk_scheduler_t *scheduler, pcmk__output_t *out,
      * example, action configuration), so we aren't necessarily checking those.
      */
     if (*cib_object != NULL) {
-        /* The scheduler takes ownership of the XML object and potentially
-         * frees it later. We want the caller of pcmk__verify to retain
-         * ownership of the passed-in XML object, hence we pass in a copy
-         * to the scheduler.
-         */
-        scheduler->input = pcmk__xml_copy(NULL, *cib_object);
+        pcmk_scheduler_t *scheduler = pcmk_new_scheduler();
```

Comment:
@clumens This commit will probably be gone by the time you look at the PR, since it breaks things. But it's an interesting side effect of unsetting `pcmk__config_has_{error,warning}` when we free the scheduler. The caller of `pcmk__verify()` doesn't need the scheduler object at all, so this purpose of this commit was to drop the argument and create a scheduler object within `pcmk__verify()` instead -- and then free it before returning.

Problem is, that unsets `pcmk__config_has_{error,warning}`, so the caller thinks there were no errors.

It's kind of odd that those are globals rather than part of the scheduler data, really... It might be that way to work around somethiing else.